pysakoinnin_sahk_asiointi/urls.py

```diff
@@ -6,11 +6,11 @@
 
 from api.api import router as api_router
 
-api = NinjaAPI()
+api = NinjaAPI(title='Pysaköinnin Sähköinen Asiointi', version='1.0.0')
```

Comment:
```suggestion
api = NinjaAPI(title='Pysäköinnin asiointi', version='1.0.0')
```
This could also be same as elsewhere :)